test/e2e/scheduling/priorities.go

```diff
@@ -217,7 +217,12 @@ var _ = SIGDescribe("SchedulerPriorities [Serial]", func() {
 		}
 	})
 
-	ginkgo.It("Pod should be preferably scheduled to nodes pod can tolerate", func() {
+	/*
+		Release : v1.16
+		Testname: Scheduler, Pod Toleration
+		Description: Pod MUST be preferably scheduled onto the Node whose taint can be tolerated.
+	*/
+	framework.ConformanceIt("Pod should be preferably scheduled to nodes pod can tolerate", func() {
```

Comment:
I'm not sure if this test is actually proving this? it is effectively doing
- apply taint "foo" to all but one node
- schedule a pod, it lands on the one un-tainted node
- apply taint "bar" to the one node
- schedule a pod with toleration "bar", it lands on the one "bar"-tainted node

... but that's the only node it _could_ land on, we're trying to test that it _prefers_ to land on that node right? wouldn't we want to do something more like
- apply the "bar" taint to one node, leave the rest untainted
- schedule a pod with a "bar" toleration repeatedly
- it should (always? usually?) land on the the node with the "bar" taint